pulsar-testclient/src/main/java/org/apache/pulsar/testclient/DefaultMessageFormatter.java

```diff
@@ -81,21 +81,18 @@ private float _getFloatValue(float size) {
 
     private String getStringValue(float size) {
         int s = (int) size;
-        if (size == 0) {
-            size = 20;
-        };
-        String result = "";
+        StringBuilder result = new StringBuilder();
         for(int i = 0; i < s; i++) {
```

Comment:
`org.apache.commons.lang3.RandomStringUtils#randomAlphabetic(int)` can be used to generate random string.

Reply:
I agree.

Reply:
Optimized